mitodl/ocw-to-hugo
# new fields for last import

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-to-hugo/issues/456

#### What's this PR do?
This PR brings in several new fields that were added to `ocw-data-parser` in https://github.com/mitodl/ocw-data-parser/pull/165.  It also brings in some fields that we weren't bringing in from instructors to preserve the data for use in future migrations.

#### How should this be manually tested?
 - Read the readme if you've never used `ocw-to-hugo` before and ensure you are set up for S3 access
 - Put the following in `private/courses.json`:
```
{
    "courses": [
        "22-033-nuclear-systems-design-project-fall-2011",
        "18-06-linear-algebra-spring-2010",
        "20-219-becoming-the-next-bill-nye-writing-and-hosting-the-educational-show-january-iap-2015"
    ]
}
```
 - Run `node . -i private/input -o private/output -c private/courses.json --download --rm`
 - Verify that the metadata properties listed in the attached issue are present in the converted course data, with course level metadata properties in `data/course_legacy.json` and video data in video resources under `content/resources` in each course.  Cross reference the values with the parsed json downloaded into `private/input`.

### NOTE
At the moment I am setting this PR as blocked because although https://github.com/mitodl/open-discussions/pull/3502 has merged, a full `backpopulate_ocw_data` will need to be run in `open-discussions` before the parsed JSON in the `open-learning-course-data` bucket will contain the new properties.
